roles/devscripts/tasks/133_host_network.yml

```diff
@@ -23,13 +23,6 @@
     name: libvirt_manager
     tasks_from: delete_network.yml
 
-- name: Ensure the required network configurations are applied.
```

Comment:
I strongly believe this entire set of tasks i.e. `133_host_network` should be moved outside

Reply:
maybe in a follow-up - for now, I want to target minimal changes.